backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java

```diff
@@ -39,7 +40,7 @@ public void Validate(String base) throws InvalidConfigException {
   }
 
   public boolean isPostgres() {
-    return RdbDialect.equals(PostgreSQLDialect.class.getName());
+    return RdbDialect.equals(PostgreSQL82Dialect.class.getName());
```

Comment:
are you sure this doesn't change anything semantically in our code? Are there checks that will fail now with this updated class name?

Reply:
`PostgreSQL82Dialect` is a parent class of `PostgreSQLDialect`  i thing no need to worry on this
Reference: https://stackoverflow.com/questions/33725589/org-hibernate-dialect-postgresqldialect-is-deprecated